versions/e-/eigen3.json

```
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "51a0bcbf78726733fa51ad1aa15efbc2f55a499c",
```

```suggestion
      "git-tree": "471f99d976ff3d1abd774c7d025c9f78dd4e4639",
```
The value for git-tree has changed now. So you need to update it.

Or you can run ` .\vcpkg x-add-version --overwrite eigen3` to update this.